lib/app/item.ex

```diff
@@ -126,6 +137,12 @@ defmodule App.Item do
     |> PaperTrail.update(originator: %{id: Map.get(attrs, :person_id, 0)})
   end
 
+  def update_draft(%Item{} = item, attrs) do
+    item
+    |> Item.draft_changeset(attrs)
```

Comment:
Use the draft changeset which allow the text to be empty compare to the normal changeset